en/docs/includes/design/generate-api-key.md

```diff
@@ -18,15 +18,15 @@
 
 5. Click **Subscribe**.
 
-     [![Subscribe to the API](https://apim.docs.wso2.com/en/4.1.0/assets/img/learn/subscribe-to-api.png)](https://apim.docs.wso2.com/en/4.1.0/assets/img/learn/subscribe-to-api.png)
+     [![Subscribe to the API]({{base_path}}/assets/img/learn/subscribe-to-api.png)]({{base_path}}/assets/img/learn/subscribe-to-api.png)
```

Comment:
I don't think base_path works for content like this where you are pulling it into other pages. Please check and revert.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/5111/commits/29ea6fcfcb2104f29dde05839dc32297d6ac56dc